ordering-client-app/src/index.tsx

```diff
@@ -1,13 +1,20 @@
 import * as React from 'react';
-import { ReactElement } from 'react';
-import { render } from 'react-dom';
+import ReactDOM from 'react-dom/client';
+import {StrictMode} from 'react';
+import { App } from './components/App';
+import 'bootstrap/dist/css/bootstrap.min.css';
 
-const App = (): ReactElement => (
-    <>
-        <h1>Hello world</h1>;
-    </>
-)
+const rootElement = document.getElementById('root');
 
-const mainDiv = document.querySelector('#main');
+if (rootElement) {
+    const root = ReactDOM.createRoot(rootElement);
 
-render(<App/>, mainDiv);
\ No newline at end of file
+    root.render(
+        // Only for dev purposes
+        <StrictMode>
+            <App />
+        </StrictMode>
+    )
+} else {
```

Comment:
If you check for negation of this condition and throw error you can gry rid of `else` here. Code will be cleaner.